javascript/javascript1/week1/index.js

```diff
@@ -0,0 +1,110 @@
+// Javascript warm up part one
+
+const myProfile = 'https://www.freecodecamp.org/kristinagru';
+console.log('Here is a link to my freecodecamp profile: ' + myProfile);
+
+// Age-ify (A future age calculator)
+
+const yearOfBirth = 1996;
+const yearFuture = 2027;
+const age = yearFuture - yearOfBirth;
+
+console.log('I will be ' + age + ' years old in ' + yearFuture);
+
+// Goodboy-Oldboy (A dog age calculator)
+
+const dogYearOfBirth = prompt('What is your dogs year of birth?');
+const dogYearFuture = 2027;
+const dogYears = dogYearFuture - dogYearOfBirth;
+const shouldShowResultInDogYears = prompt(
+  'Would you like the result to be shown in dog years? Type either "Yes" or "No"'
+).toLowerCase();
```

Comment:
Nice work with prompts, however, if you want to collect users input, you should be aware that the user could type in anything and it is our responsibility to check. prompt() method returns string, so, could you please check that user has actually submitted a number?